sync-core/src/test/java/com/cloudant/common/CouchTestBase.java

```diff
@@ -25,7 +29,10 @@ public CouchConfig getCouchConfig(String db) {
                         " is required to run tests with Cloudant.");
             }
             return config;
-        } else {
+        } else if (SPECIFIED_COUCH) {
+            return SpecifiedCouch.defaultConfig(db);
```

Comment:
Can we replace the `TEST_WITH_CLOUDANT` stuff with `SPECIFIED_COUCH`? It seems that they're both doing the same job, that is allowing the user to configure something other than a localhost-ish couchdb (just CloudantConfig is a bit more specific)?